docs/governance/overview.md

```diff
@@ -1,26 +1,26 @@
 ---
 id: overview
 slug: /governance/
-title: Participate in Governance 
+title: Participate in Governance
 sidebar_label: Overview
-description: Governance is an essential topic that includes Cardano Improvement Proposals (CIP), project funding, voting, and network parameters.
+description: Governance is an essential topic that includes the on-chain governance framework, Cardano Improvement Proposals (CIP), Cardano Problem Statements (CPS), Project Catalyst, and Network Parameters.
```

Comment:
```suggestion
description: Governance is an essential topic that includes the on-chain protocol governance, Cardano Improvement Proposals (CIP), Cardano Problem Statements (CPS), Project Catalyst, and Network Parameters.
```
I think "protocol governance" is a more specific term

Reply:
I like it. Thanks for the suggestion.